### PR TITLE
Fixed link field when UUIDs are switched off

### PR DESCRIPTION
**Pull request by bastianallgeier**
## Fixes 

- The link field is now usable when UUIDs are switched off. The page and file browsers will insert the absolute URL instead of the UUID https://github.com/getkirby/kirby/issues/5489

### CODE REVIEW

**Comment by distantnative** [comment]
aria-select doesn't work in PageTree anymore

File: panel/src/components/Forms/Field/LinkField.vue [hunk 1]
```diff
@@ -345,6 +349,24 @@ export default {
...
+		},
+		selectFile(file) {
+			if (file.uuid) {
+				this.onInput(file.uuid);
+				return;
+			}
+
+			this.switchType("url");
+			this.onInput(file.url);
+		},
```
Comment (afbora) [inline panel/src/components/Forms/Field/LinkField.vue]: This code block can be simplify like that

```suggestion
		selectModel(model) {
			if (model.uuid) {
				this.onInput(model.uuid);
				return;
			}

			this.switchType("url");
			this.onInput(model.url);
		},
```

**Comment by afbora** [comment]
Minor issue on linking to `site`:

https://github.com/getkirby/kirby/assets/3393422/9efc0c30-d290-4cb7-8741-5e8cb55afab0

**Comment by bastianallgeier** [comment]
@getkirby/frontend 

- fixed aria-selected in the tree
- fixed site link

**Review by afbora — APPROVED** [review]
Looks great to me 🎉

**Pull request merged**